contact.html

```diff
@@ -0,0 +1,245 @@
+                <input type="checkbox" class="input-hidden form-switcher__input" id="switch">
+                <label for="switch" class="form-switcher__box"></label>
+            </div>
+            <div class="form-control">
+                <label class="form-control__label" for="name">Name</label>
+                <input type="text" id="name" class="form-control__input" minlength="3" required>
+            </div>
+            <div class="form-control">
+                <label class="form-control__label" for="lastname">Last name</label>
+                <input type="text" id="lastname" class="form-control__input" required>
+            </div>
+            <div class="form-control">
+                <label class="form-control__label" for="date-of-birth">Date of birth</label>
+                <input type="date" value="dd/mm/yyyy" id="date-of-birth" class="form-control__input" required>
+            </div>
+            <div class="form-control">
+                <div class="form-control__label" for="date-of-birth">Gender</div>
+                <label class="form-radio">
+                <input class="input-hidden form-radio__input" type="radio" name="gender" checked required>
+                <span class="form-radio__inner">
+                    <span class="form-radio__box"></span>
+                    <span class="form-radio__text">Male</span>
+                </span>
+                </label>
+                <label class="form-radio">
+                <input class="input-hidden form-radio__input" type="radio" name="gender" value="female">
+                <span class="form-radio__inner">
+                    <span class="form-radio__box"></span>
+                    <span class="form-radio__text">Female</span>
+                </span>
+                </label>
+            </div>
+            <div class="form-control">
+                <label class="form-control__label" for="email">Email</label>
+                <input type="email" id="email" class="form-control__input" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required>
+            </div>
+            <div class="form-control">
+                <label class="form-control__label" for="phone">Phone number</label>
+                <input type="tel" id="phone" class="form-control__input" required>
+            </div>
+            <div class="form-control">
+                <label class="form-control__label" for="message">Message</label>
+                <textarea id="message" class="form-control__input"></textarea>
+            </div>
+            <label class="form-checkbox">
+                <input type="checkbox" class="input-hidden form-checkbox__input" required>
+                <p class="form-checkbox__inner">
+                <span class="form-checkbox__box"></span>
+                <span class="form-checkbox__text">By clicking on the ‘Send Message’ button I agree to <a target="_blank">all the terms of the security policy</a></span>
```

Comment:
Please add href attribute for `<a target="_blank">all the terms...`

Reply:
done